source/common/config/protobuf_link_hacks.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#ifndef PROTOBUF_LINKS_
+#define PROTOBUF_LINKS_
```

Comment:
I see, for coverage this is broken as they are both in the global namespace. What if you use these protos in local vars in an `inline` method that is defined here, then instantiate the function in the various modules? That should avoid introducing any globals?

I'm not sure how PROTOBUF_LINKS_ define works here, since #pragma once is a similar thing.